.github/workflows/release.yml

```diff
@@ -0,0 +1,20 @@
+name: Create Release
```

Comment:
- [ ] Let's add an item to the CHANGELOG.md for this - under `Minor changes`: Converted CI from Travis to Github Actions and added CD

Reply:
Updated CHANGELOG.md file as suggested.